platform/feature-api/communication/src/remote-pod/tests/node/bubblewrap.test.ts

```diff
@@ -0,0 +1,19 @@
+import { podBubblewrapClasses, podBubblewrap } from "../../remote";
+import { NamedNode } from "@polypoly-eu/api";
+
+describe("Classes are accounted for", () => {
+    it("has the right number of classes", () => {
+        expect(Object.keys(podBubblewrapClasses).length).toBe(6);
+    });
+});
+
+describe("Classes roundtrip correctly", () => {
+    it("works for NamedNode", () => {
+        const namedNode = new NamedNode("https://example.org/n");
+        const encoded = podBubblewrap.encode(namedNode);
+        expect(encoded).toBeTruthy();
+        const decoded = podBubblewrap.decode(encoded);
+        expect(decoded).toBeInstanceOf(NamedNode);
+        expect(decoded).toStrictEqual(namedNode);
+    });
+});
```

Comment:
```suggestion
describe("Bubblewrap", () => {

	  describe("when create()", () => {
	      test("should have the right number of classes", () => {
	          expect(Object.keys(podBubblewrapClasses).length).toBe(6);
	      });
	  });

	  describe("when NamedNode() is called", () => {
	      test("should generate roundtrip classes correctly", () => {
	          const namedNode = new NamedNode("https://example.org/n");
	          const encoded = podBubblewrap.encode(namedNode);
	          
	          expect(encoded).toBeTruthy();
	          
	          const decoded = podBubblewrap.decode(encoded);
	          
	          expect(decoded).toBeInstanceOf(NamedNode);
	          expect(decoded).toStrictEqual(namedNode);
	      });
	  });
});

```

Reply:
What's the difference here? Functionally it's still the same, right?

Reply:
yep a bit better writing of the test only...